src/test/java/com/iexec/worker/docker/DockerClientServiceTests.java

```diff
@@ -628,6 +627,30 @@ public void shouldNotRemoveContainerSinceDockerCmdException() {
         assertThat(dockerClientService.removeContainer(getRandomString())).isFalse();
     }
 
+    @Test
+    public void shouldRemoveRegistryUrlFromImageName() {
+        assertThat(
+                dockerClientService
+                        .normalizeImageName("registry/namespace/image:tag")
+        ).isEqualTo("namespace/image:tag");
+    }
+
+    @Test
+    public void shouldAcceptImageNameWithoutRegistryUrl() {
+        assertThat(
+                dockerClientService
+                        .normalizeImageName("namespace/image:tag")
+        ).isEqualTo("namespace/image:tag");
+    }
+
+    @Test
+    public void ShouldAcceptOfficialImageName() {
+        assertThat(
+                dockerClientService
+                        .normalizeImageName("official:tag")
+        ).isEqualTo("official:tag");
+    }
+
```

Comment:
Is `namespace/image` working?

Reply:
No it does not work with the current filter.

Reply:
I think about things like this: `docker.io/alice/offchain-tee-kaiko-pricefeed` which should pull `latest`

Reply:
Yes we need to make some update to support it if we want. But I think it is not a good idea. We shouldn't accept images without tags IMHO.

Reply:
I agree, I'm just thinking about supporting old apps